framework_tvb/tvb/interfaces/rest/client/examples/launch_operation.py

```diff
@@ -95,12 +91,13 @@
     surface_view_model.uploaded = surface_zip_path
     surface_view_model.surface_type = "Cortical Surface"
     surface_view_model.should_center = False
+    file_name = ZIPSurfaceImporterForm.get_upload_information().keys()
+    data_files = {list(file_name)[0]: surface_zip_path}
 
     logger.info("Launching surface uploading operation...")
     algorithm_module = 'tvb.adapters.uploaders.zip_surface_importer'
     algorithm_class = 'ZIPSurfaceImporter'
-    operation_gid = tvb_client.launch_operation(project_gid, algorithm_module, algorithm_class, surface_view_model,
-                                                data_file_path_1=surface_zip_path)
+    operation_gid = tvb_client.launch_operation(project_gid, algorithm_module, algorithm_class, surface_view_model, data_files=data_files)
```

Comment:
It is ok now, in terms of how generic the code was written, but the API is hard to use, with some redundant information requested from the end-user.
I would like this to be the end call:
`operation_gid = tvb_client.launch_operation(project_gid, ZIPConnectivityImporter, connectivity_view_model)`
And we should deal with all the strings and files selection ourselves